core/src/com/po/fuck/enemies/BasicEnemy.java

```diff
@@ -27,21 +29,31 @@ public final class BasicEnemy extends Entity implements Updatable {
 
     public BasicEnemy(Vector2 position) {
         super(position, 5);
+        this.teamTag = Constants.CREEP_TEAM_TAG;
     }
 
     @Override
     public void update(float delta) {
-        Player player = FUCK.player.get();
-        if (player == null)
+        ArrayList<Entity> opponents = All.entityCollection.getOpponents(this.teamTag);
+        if (opponents == null) {
             return;
+        }
+        Entity target = opponents.get(0);
+        float dist = GeometryMisc.distance(this, target);
```

Comment:
I think this code should be moved into `GeometryMisc`

Reply:
You mean lines 43-48?

Reply:
Finding closest enemy lines 41-48

Reply:
It has been moved, but wrong